tests/core/test_ensemble.py

```diff
@@ -377,3 +380,64 @@ def test_from_dict_does_not_change_passed_dict_parameter():
     PolicyEnsemble.from_dict(config_copy)
 
     assert config == config_copy
+
+
+def test_rule_based_data_warnings_no_rule_trackers():
+    trackers = [DialogueStateTracker("some-id", slots=[], is_rule_tracker=False)]
+    policies = [RulePolicy()]
+    ensemble = SimplePolicyEnsemble(policies)
+
+    with pytest.warns(UserWarning) as record:
+        ensemble.train(trackers, Domain.empty(), RegexInterpreter())
+
+    assert (
```

Comment:
optional: I think it's enough to just test for a warning and not for the actual string

Reply:
compare this test with the one below - `train()` emits two different warning messages and these two tests are specifically about making sure that the correct one is emitted in each case